frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card-table/feature-flag-root-section-card-table.component.html

```diff
@@ -33,7 +33,14 @@
         {{ FLAG_TRANSLATION_KEYS.STATUS | translate }}
       </th>
       <td mat-cell *matCellDef="let flag" class="status-column ft-14-400">
-        <app-common-status-indicator-chip [chipClass]="flag.status"></app-common-status-indicator-chip>
+        <app-common-status-indicator-chip
+          [chipClass]="flag.status"
+          [showWarning]="
+            flag.status === FEATURE_FLAG_STATUS.ENABLED &&
```

Comment:
this logic is duplicated in both components, can also easily be made a selector. That would be my ngrx recommended approach but I'll leave it up to you to decide if you want to do that.